scripts/dss-start-stop--think-before-you-invoke-me.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+"""
+Use the AWS Lambda concurrency property to disable/enable lambda functions:
+    - set concurrency to 0 to disable execution
+    - remove concurrency setting to enable execution
+"""
+import os
+import sys
+import boto3
+import click
+import argparse
+
+
+stage = os.environ['DSS_DEPLOYMENT_STAGE']
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("--stop", action="store_true")
+parser.add_argument("--start", action="store_true")
+args = parser.parse_args()
+assert(
+    (args.start or args.stop) and not (args.start and args.stop)
+)
+
+
+action = f"DSS {stage} START" if args.start else f"DSS {stage} STOP"
+if not click.confirm(f"Are you sure you want to do this ({action})?"):
+    sys.exit(0)
+
+
+LAMBDA = boto3.client('lambda')
```

Comment:
All-capsing should be reserved for variables with a constant intializer. It is not evident why this variable is all-caps while `action`, `args`, `stage` etc. are not.

Reply:
changed